deploy/crds/smartgateway.infra.watch_v2alpha1_smartgateway.metrics_cr.yaml

```diff
@@ -7,4 +7,4 @@ spec:
   debug: false
   serviceType: "metrics"
   useTimestamp: true
-  prefetch: 15000
+  amqpDataSource: collectd
```

Comment:
We don't plan to collect ceilometer metrics?

Reply:
It does! That is just one of the example CRs